pkg/ring/kv/dynamodb/metrics.go

```diff
@@ -29,8 +29,8 @@ func newDynamoDbMetrics(registerer prometheus.Registerer) *dynamodbMetrics {
 	}, []string{"operation", "status_code"}))
 
 	dynamodbUsageMetrics := promauto.With(registerer).NewCounterVec(prometheus.CounterOpts{
-		Name: "dynamodb_kv_read_capacity_total",
-		Help: "Total used read capacity on dynamodb",
+		Name: "dynamodb_kv_read_write_capacity_total",
```

Comment:
Should we add a changelog for this?
I think the problem is that we are changing the name of the metric and if someone is already using it, it might break their usecase.
Also how about `dynamodb_kv_consumed_capacity_total`. Might be more generic for future operations.

Reply:
makes sense. I will make the change and add statement in ChangeLog.